test/test_broker/acceptance/service-fabrik-api-2.0.instances.director.spec.js

```diff
@@ -1964,7 +1964,6 @@ describe('service-fabrik-api-sf2.0', function () {
             space_guid: space_guid,
```

Comment:
@muralisc : same in this class too , pls remove commented code.

Reply:
Done